composer.json

```diff
@@ -27,17 +27,17 @@
         "sonata-project/cache": "^1.0 || ^2.0",
         "sonata-project/doctrine-extensions": "^1.10.1",
         "sonata-project/form-extensions": "^0.1.1 || ^1.4",
-        "symfony/asset": "^4.4",
-        "symfony/config": "^4.4",
+        "symfony/asset": "^4.4 || ^5.2",
+        "symfony/config": "^4.4 || ^5.2",
         "symfony/console": "^4.4",
-        "symfony/dependency-injection": "^4.4",
+        "symfony/dependency-injection": "^4.4 || ^5.2",
         "symfony/event-dispatcher": "^4.4",
         "symfony/form": "^4.4",
         "symfony/framework-bundle": "^4.4",
-        "symfony/http-foundation": "^4.4",
+        "symfony/http-foundation": "^4.4 || ^5.2",
         "symfony/http-kernel": "^4.4",
-        "symfony/options-resolver": "^4.4 || ^5.1",
-        "symfony/templating": "^4.4",
+        "symfony/options-resolver": "^4.4 || ^5.2",
+        "symfony/templating": "^4.4 || ^5.2",
         "symfony/twig-bundle": "^4.4",
```

Comment:
What about this?

Reply:
The constraint I left intact are those that belong to dependencies that can't be resolved to version ^5.2.
In this case:
```
composer why-not symfony/twig-bundle:^5.2

sonata-project/admin-bundle  3.91.1     requires          symfony/twig-bundle (^4.4)                      
symfony/twig-bundle          5.2.x-dev  conflicts         symfony/framework-bundle (<5.0)                 
symfony/twig-bundle          5.2.x-dev  conflicts         symfony/translation (<5.0)                      
symfony/twig-bundle          5.2.x-dev  requires          symfony/twig-bridge (^5.0)                      
sonata-project/block-bundle  3.x-dev    does not require  symfony/twig-bridge (but v4.4.20 is installed)  
symfony/twig-bundle          5.2.x-dev  requires          symfony/http-kernel (^5.0)                      
sonata-project/block-bundle  3.x-dev    requires          symfony/http-kernel (^4.4)
```